### PR TITLE
Created readme

### PR DESCRIPTION
**Pull request by chrisdolendo**

### CODE REVIEW

**Comment by krevels** [comment]
This looks good.  I've pointed out a handful of typos, though, which you can please fix when you have a chance

File: README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#Merchant Script
+
+## Setup instructions:
+
+1.  Create a `.env` file using `.end.sample` as a template.
```
Comment (krevels) [inline README.md]: .env.sample

File: README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  - OUTPUT_FILE and BUILD_DIR should correspond to src path in index.html for the spare-mount.
```
Comment (krevels) [inline README.md]: sparo-mount

File: README.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  -  `IFRAME_SRC` source points to charity selection widget (either local or hosted)
+  - `TXAPI_URL` points to charity api node (either local or hosted)
+
+2. Run `npm install` and `npm start`
+3. Go to `localhost:8080`
```
Comment (krevels) [inline README.md]: Go to `localhost:<configured port number>`

**Comment by chrisdolendo** [comment]
@krevels fixed the typos!

**Pull request merged**